python/triton/third_party/hip/hip_backend.py

```diff
@@ -317,11 +324,12 @@ def get_amdgpu_arch_fulldetails():
         gfx_arch = os.environ.get('MI_GPU_ARCH', arch_name)
         if gfx_arch is None:
             raise RuntimeError('gfx_arch is None (not specified)')
-        mat_core_ver = gpu_matrix_core_version()
-        capability = gpu_matrix_core_version() * 100
+        mat_core_ver = gpu_matrix_core_version(arch_name)
```

Comment:
Is this change strictly required for configurable warp size? It seems to be an orthogonal change that fixes the behavior of `MI_GPU_ARCH`
(Furthermore I don't think this env var is well named, I would use `TRITON_OVERRIDE_GPU_ARCH` or sth similar)